setup.py

```diff
@@ -112,6 +112,7 @@
     ],
     data_files=[
         ('/etc/bash_completion.d', glob.glob('data/etc/bash_completion.d/*')),
+        ('/usr/share/sonic/templates/sonic-environment.j2', 'sonic_installer/templates/sonic-environment.j2'),
```

Comment:
should this template file reside in sonic-utilities or reside in sonic-buildimage?

Reply:
sonic-utilities looks alright. because this file is used by sonic-installer